docs/migrate-from/ibm-watson-to-rasa.rst

```diff
@@ -27,5 +27,5 @@ At Rasa, we hear a few different reasons why developers switch from cloud-based
 
 
 .. button::
-     :link: ../get_started_step1/
+     :link: ../../user-guide/rasa-tutorial.html
```

Comment:
?I think this should probably go to the getting started page, what do you think?

Reply:
oh, no I wasn't sure what `get_started_step1` used to be, so I picked the first thing I saw =P